### PR TITLE
BZ#1293820 - Updated Integrating External Service section

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For [BZ#1293820](https://bugzilla.redhat.com/show_bug.cgi?id=1293820)

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bfallonf peer review please

File: dev_guide/integrating_external_services.adoc [hunk 1]
```diff
@@ -55,7 +55,7 @@ requests to the service amongst the service's endpoints.
 Services that represent an external resource do not require associated pods.
 Instead, leave the `Selector` field unset. This represents the external service,
 making the `EndpointsController` ignore the service and allows you to specify
-endpoints manually:
+endpoints manually using IP address:
```
Comment (bfallonf) [inline dev_guide/integrating_external_services.adoc]: You've added "using IP address" above, but the example below doesn't have an IP address anywhere. Should it?

**Comment by bfallonf** [comment]
@gaurav-nelson Stuff looks good, but I've added a question.

File: dev_guide/integrating_external_services.adoc (hunk 1 above)
Comment (gaurav-nelson) [inline dev_guide/integrating_external_services.adoc]: Thanks @bfallonf I'll check and update.

**Comment by gaurav-nelson** [comment]
@bfallonf Can you take another look at this? I have restructured the text.

File: dev_guide/integrating_external_services.adoc [hunk 2]
```diff
@@ -113,6 +119,27 @@ value in the service defined in the previous step.
 
 ====
 
+==== Using FQDN
+Use an ExternalName service without defining any ports or endpoints.
```
Comment (bfallonf) [inline dev_guide/integrating_external_services.adoc]: I'd say `ExternalName` should be marked up?

File: dev_guide/integrating_external_services.adoc [hunk 3]
```diff
@@ -326,6 +355,26 @@ use the service by setting environment variables in the appropriate containers:
...
```
Comment (bfallonf) [inline dev_guide/integrating_external_services.adoc]: Same here.

**Comment by bfallonf** [comment]
@gaurav-nelson Aside from the comment above, I'd an anchors to the headings. But other than that, I think this works :+1:

**Comment by gaurav-nelson** [comment]
Thanks @bfallonf fixed those issues. Merging now.

**Comment by adellape** [comment]
> Aside from the comment above, I'd an anchors to the headings. But other than that, I think this works

@gaurav-nelson I think @bfallonf's comment about needing anchor unique IDs may have been missed. Just a reminder:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#unique-ids

**Pull request merged**